fplus-lib/src/core/autoallocator/file.rs

```diff
@@ -0,0 +1,10 @@
+use alloy::primitives::Address;
+use serde::{Deserialize, Serialize};
+
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub struct Autoallocation {
```

Comment:
What's this struct for, I don't see it used.
Also, why this file is named `file.rs`?